edgelet/hsm-sys/azure-iot-hsm-c/src/edge_openssl_common.c

```diff
@@ -0,0 +1,17 @@
+#include <openssl/err.h>
+#include <openssl/x509.h>
+#include "azure_c_shared_utility/gballoc.h"
+#include "edge_openssl_common.h"
+
+void initialize_openssl(void)
+{
+    static bool is_openssl_initialized = false;
```

Comment:
OpenSSL initialization is global, right? Not per handle or something like that? This function will be scoped to the process rather than to some "session" within a process...I just want to double-check that's what was intended.

Reply:
Once per process.

Reply:
👌